apps/src/dom.js

```diff
@@ -30,7 +30,10 @@ var addEvent = function(element, eventName, handler) {
   var touchEvent = exports.getTouchEventName(eventName);
   if (touchEvent) {
     element.addEventListener(touchEvent, function(e) {
-      e.preventDefault();  // Stop mouse events.
+      // Stop mouse events and suppress default event handler to prevent
+      // unintentional double-clicking
+      e.preventDefault();
+      element.removeEventListener(eventName, handler);
```

Comment:
I was wondering why this wasn't more of a problem everywhere; I could certainly use debounce to prevent the specific bug I was experiencing because of this, but I wanted to figure out why it was happening in the first place.

Actually, if this works, I might be able to go through and remove some of those debounces.